website/pages/en/glossary.mdx

```diff
@@ -81,3 +81,7 @@ title: Glossary
 - **Cooldown Period**: The time remaining until an Indexer who changed their delegation parameters can do so again.
 
 - **L2 Transfer Tools**: Smart contracts and UI that enable network participants to transfer from Ethereum mainnet to Arbitrum One. Network participants can transfer delegated GRT, subgraphs, curation shares, and Indexer's self stake.
+
+- **"Upgrading" a subgraph to The Graph Network**: The process of moving a subgraph from the hosted service to The Graph Network.
+
+- **"Updating" a subgraph**: The process of updating a subgraph's schema and/or mappings, resulting in a new subgraph version.
```

Comment:
Could we use italics instead of quotes?

```suggestion
- **_Upgrading_ a subgraph to The Graph Network**: The process of moving a subgraph from the hosted service to The Graph Network.

- **_Updating_ a subgraph**: The process of updating a subgraph's schema and/or mappings, resulting in a new subgraph version.
```

Reply:
Yes, works for me! Thank you!